analytics/views.py

```diff
@@ -1214,6 +1215,10 @@ def support(request: HttpRequest) -> HttpResponse:
                     realm.current_plan.licenses = last_ledger_entry.licenses
                     realm.current_plan.licenses_used = get_latest_seat_count(realm)
 
+        # full_names can have , in them
+        users.update(UserProfile.objects.filter(full_name__iexact=query))
+
+        context["users"] = users
```

Comment:
This is a `set`—should these be sorted by something?

Reply:
I don't think sorting is necessary. But @mateuszmandera should know better since he is the one who primarily uses this page.

Reply:
Yeah I don't think ordering matters here